src/main/java/fivium/pat/graphql/queryfields/superuser/DeleteClinicianQF.java

```diff
@@ -23,7 +23,7 @@
 
 public class DeleteClinicianQF extends PAT_BaseQF {
 
-	private static final String DELETE_CLINICIAN_PREPARED_SQL_QUERY = "DELETE FROM rns_internal.clinicians WHERE Email=?;";
+	private static final String DELETE_CLINICIAN_PREPARED_SQL_QUERY = "DELETE FROM clinicians WHERE Email=?;";
```

Comment:
Should also tidy up fetchData(..) method so it only has 1 try/catch and doesn't return the server specific error to the client.  See CliniciansChangePasswordQF for an example